src/openrct2/ride/Vehicle.cpp

```diff
@@ -6393,26 +6396,32 @@ int32_t Vehicle::UpdateMotionDodgems()
  *
  *  rct2: 0x006DD365
  */
-bool vehicle_update_dodgems_collision(Vehicle* vehicle, int16_t x, int16_t y, uint16_t* spriteId)
+static bool wouldCollideWithDodgemsTrackEdge(
+    CoordsXY coords, CoordsXY trackLocation, uint32_t trackType, uint16_t dodgemsCarRadius)
```

Comment:
```suggestion
    const CoordsXY& coords, cons CoordsXY& trackLocation, uint32_t trackType, uint16_t dodgemsCarRadius)
```
(formatting will probably need to be revisited as well)